CONTRIBUTING.md

```diff
@@ -6,32 +6,32 @@ We welcome community contributions to vcf-validator! To build tool locally, foll
+Once you found an interesting issue to work on you may check if someone other is not working on the same issue. You can ask once in the issue to get that issue assigned to you. Once you get an approval go for the PR. While making a new PR do follow these norms:
 
--  Fetch the latest changes by `git fetch upstream`
--  Update the local develop branch `git rebase upstream/develop develop`
--  Create a new branch from develop `git checkout -b <branch_name> develop`
+-  Fetch the latest changes by `git fetch upstream`.
+-  Update the local develop branch `git rebase upstream/develop develop`.
+-  Create a new branch from develop `git checkout -b <branch_name> develop`.
 -  Add and commit the changes, commit messages should be short and well explanatory.
 -  While creating new PR, change the base from master to develop.
 -  PR message should be well explanatory and should contain reference to the issue this PR is for.
 -  Once the PR is ready for a review, ask for a review.
 
 ## Coding Practice
-There are some coding practices that we follow to make code easily readable and modifiable. few of them that should be kept in mind while coding are:
+There are some coding practices that we follow to make code easily readable and modifiable. Few of them that should be kept in mind while coding are:
 
-- Consistent Indentation - we generally prefer 4 spaces for indentation
-- Comments - provide comments whenever required, so that it may be easy for other developers to read your code. Also avoid obvious comments.
-- Tests - add tests if new functionality is added to ensure that this functionality doesn't break in future.
-- Consistent Naming Scheme - follow a consistent naming scheme. we prefer `snake_case`(underscored) instead of `camelCase`. naming of variables should be done in well explanatory way.
-- DRY Principle - Every piece of knowledge must have a single, unambiguous, authoritative representation within a system.
-- Avoid deep nesting - too many levels of nesting make code harder to read.
-- Limit line length - It is a good practice to avoid writing horizontally long lines of code.
-- Bracket style - we generally use attached brackets for `conditions` and `loops` and broken brackets for `functions` and `classes`.
+- Consistent Indentation - We generally prefer 4 spaces for indentation.
+- Comments & Documentation - Provide comments whenever required, so that it may be easy for other developers to read your code. Also avoid obvious comments.
+- Tests - Add tests if new functionality is added to ensure that this functionality does not break in future.
+- Consistent Naming Scheme - Follow a consistent naming scheme. we prefer `snake_case`(underscored) instead of `camelCase`. naming of variables should be done in well explanatory way.
```

Comment:
put "we" and "naming" in uppercase

Reply:
I think I strongly need to write a script to detect a lowercase letter after a `dot`. :sweat_smile: